### PR TITLE
Test improvements

### PR DESCRIPTION
**Pull request by crepererum**
This fixes multiple issues I've found during dogfooding.

### CODE REVIEW

File: {{ cookiecutter.project_shortname }}/setup.py [hunk 1]
```diff
@@ -10,13 +10,16 @@
 readme = open('README.rst').read()
 history = open('CHANGES.rst').read()
 
+invenio_version = '>=2.0.3'
+
 requirements = [
     'Flask>=0.10.1',
     'six>=1.7.2',
-    'Invenio>=2.0.3',
+    'Invenio{}'.format(invenio_version),
```
Comment (jirikuncar) [inline {{ cookiecutter.project_shortname }}/setup.py]: this should be removed as none of the packages should depend on *Invenio*. There will be *Flask-AppFactory* and *Invenio-Ext(ensions)* packages ...

Comment (crepererum) [inline {{ cookiecutter.project_shortname }}/setup.py]: What is the timeline for that one?

Comment (jirikuncar) [inline {{ cookiecutter.project_shortname }}/setup.py]: yesterday ...

Comment (crepererum) [inline {{ cookiecutter.project_shortname }}/setup.py]: oÔ

**Comment by crepererum** [comment]
Rebased against #16

**Comment by lnielsen** [comment]
Superseded by #25

**Pull request closed without merging**